### ISSUE TITLE
Update go-log to restore ndjson logging

### ISSUE THREAD

**Issue by Stebalien** [ipfs/go-ipfs#5709]
We changed go-log to spit out pretty json but we need to change it back as, apparently js-ipfs-api relies on it spitting out ndjson.

Unfortunately, as we're trying to cut a release, we didn't want to bubble the go-log change (pulls in *every* package). Therefore, we've just modified the `log tail` command to reformat the output. Once we've bubbled the go-log update, we need to remove the extra code in core/commands/log.go.

**Comment by kjzz** [comment]
You mean that we should encode log like ndjson?
```
{ "event": "Bitswap.Rebroadcast.idle", "session": "799c2a54-4bb2-4f9f-984c-c951b3dfb017", "system": "bitswap", "time": "2018-10-31T15:25:53.097215Z"}
```

**Comment by Stebalien** [comment]
Yes. This has actually ben fixed (https://github.com/ipfs/go-log/pull/53) we just need to pull the update through to go-ipfs. Unfortunately, doing so will drag along other changes in go-libp2p that we don't want to pull in before the release. We're now running the latest go-ipfs on one of our gateways to try to hammer out any bugs and we don't want to risk introducing *new* bugs by updating too much stuff.

**Comment by kjzz** [comment]
If you don't work for this now，maybe i can have a try.

**Comment by Stebalien** [comment]
Really, all I need to do is run a single command and push a bunch of changes to a bunch of repos. It'll be much easier if I just do it (once we've cut a release).

**Comment by kjzz** [comment]
Ok,If anything else i can help , please ping me.Thx a lot

**Comment by magik6k** [comment]
@Stebalien can you try also pulling update for https://github.com/gxed/sha256-simd with this to fix an error in https://github.com/ipfs/go-ipfs/issues/5694 (WASM support)?

**Comment by Stebalien** [comment]
Will do.
